src/structures/Player.ts

```diff
@@ -139,15 +146,37 @@ export class Player {
   ): Promise<SearchResult> {
     return this.manager.search(query, requester);
   }
+  /**
+   * Move the player to another connected node
+   * @param name
+   */
+  async movePlayer(name: string) {
+    const node = this.manager.nodes.get(name)
+    if(!node.connected) throw Error('The node is not connected');
+    if (!name || node.options.identifier === this.node.options.identifier) return this;  
+    const options = {
+      op: "play",
+      guildId: this.guild,
+      track: this.queue.current.track,
+      startTime: this.position,
+    };
 
+    if (typeof options.track !== "string") {
+      options.track = (options.track as Track).track;
+    }
+    this.node = node;
+    this.manager.players.set(this.guild, this)
+    await this.node.send({ op: "voiceUpdate", guildId: this.voiceState.guildId, sessionId: this.voiceState.sessionId, event: { token: this.voiceState.event.token, endpoint: this.voiceState.event.endpoint } })
```

Comment:
You can use the variable `Player#voiceState` ([type](https://github.com/MenuDocs/erela.js/blob/master/src/structures/Utils.ts#L322), [usage](https://github.com/MenuDocs/erela.js/blob/master/src/structures/Manager.ts#L457-L463)) instead as it has all the info required to minimize the use of named properties